openforcefield/topology/molecule.py

```diff
@@ -2991,9 +3021,9 @@ def to_file(self,
         --------
 
         >>> molecule = Molecule.from_iupac('imatinib')
-        >>> molecule.to_file('imatinib.mol2', outfile_format='mol2')
-        >>> molecule.to_file('imatinib.sdf', outfile_format='sdf')
-        >>> molecule.to_file('imatinib.pdb', outfile_format='pdb')
+        >>> molecule.to_file('imatinib.mol2', outfile_format='mol2')  # doctest: +SKIP
```

Comment:
Hm, so it looks like it makes it into the docs. Not a huge deal, but something we can fix later: https://open-forcefield-toolkit.readthedocs.io/en/add-doctests/api/generated/openforcefield.topology.Molecule.html#openforcefield.topology.Molecule.to_file

Opened this in #230

Reply:
I see. I was kind of hoping they didn't or that there was an option to disable them. I can actually remove the skip directive in this case if you prefer. The code actually works. I added the directives only to avoid creating those files when doctests are run locally, but maybe it doesn't matter if they run on the Travis virtual machine.

Reply:
This won't be the only doctest test we may want to skip. I'd be OK to leave it in as motivation to find a cleaner way.